recipes/clhpp/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+cp "${RECIPE_DIR}/LICENSE.txt" "${SRC_DIR}/LICENSE.txt"
```

Comment:
FYI the license should be in the next release. It is already in `master`.

xref: https://github.com/KhronosGroup/OpenCL-CLHPP/pull/21